apps/server/lib/one-drive/src/internal/utils/values.ts

```
@@ -56,14 +59,14 @@ export const coerceInteger = (value: unknown): number => {
 	if (typeof value === "number") return Math.round(value);
 	if (typeof value === "string") return parseInt(value, 10);
 
-	throw new GoogleDriveError(`Could not coerce ${value} (type: ${typeof value}) into a number`);
+	throw new OneDriveError(`Could not coerce ${value} (type: ${typeof value}) into a number`);
 };
 
 export const coerceFloat = (value: unknown): number => {
 	if (typeof value === "number") return value;
 	if (typeof value === "string") return parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)